app/src/pages/header/components/AddMovieForm.jsx

```diff
@@ -18,25 +20,40 @@ const ColoredButtonWrapper = styled(ColoredButton)`
 `
 
 function AddMovieForm(props) {
-    const [genres, setGenres] = useState([]);
-    
-    useEffect(() => {
-        Promise
-            .resolve(genresFromJson)
-            .then(genres => setGenres(genres))
-    }, [])
+    const [title, setTitle] = useState('');
+    const [releaseDate, setReleaseDate] = useState('');
+    const [url, setUrl] = useState('');
+    const [genres, setGenres] = useState('');
+    const [overview, setOverview] = useState('');
+    const [runtime, setRuntime] = useState(0);
+
+    function confirmAdd(event){
+        event.preventDefault();
+        var movie = {
+            title: title,
+            release_date: releaseDate,
+            poster_path: url,
+            genres: genres.split(','),
+            overview: overview,
+            runtime: Number.parseInt(runtime)
+        }
+
+        store.dispatch(addMovie(movie));
+        props.close();
+    }
 
     return (
         <FormComponent>
             <Title>ADD MOVIE</Title>
             <FormOptionName>TITLE</FormOptionName>
-            <FormOptionInput placeholder='Title here' />
+            <FormOptionInput placeholder='Title here' onChange={event => setTitle(event.target.value)} />
             <FormOptionName>RELEASE DATE</FormOptionName>
-            <FormOptionInput placeholder='Release date here' />
+            <FormOptionInput placeholder='Release date here' onChange={event => setReleaseDate(event.target.value)} />
             <FormOptionName>MOVIE URL</FormOptionName>
-            <FormOptionInput placeholder='Movie URL gere' />
-            <FormOptionName>GENRE</FormOptionName>
-            <FormOptionDropdown>
+            <FormOptionInput placeholder='Movie URL here' onChange={event => setUrl(event.target.value)} />
+            <FormOptionName>GENRES</FormOptionName>
+            <FormOptionInput placeholder='Genres here' onChange={event => setGenres(event.target.value)} />

```

Comment:
Do we need **event** in this line?

Reply:
Discussed) Yes, we need: we pass `event.target.value` into `setGenres`